### PR TITLE
CONTRIBUTING: fix links

### PR DESCRIPTION
**Pull request by phil-blain**
- The link to the TODO was missing the '.md' extension so it
did not work.
- The links for forking the repo and opening a PR did not work
because GitHub automatically adds '/blob/$branch' to relative links

For PRs I did not find a URL trick to automatically activate the "Compare across forks" function. Should I add a mention of this link? like this:
~~~diff
diff --git a/CONTRIBUTING.md b/CONTRIBUTING.md
index 9987e9e..308ad7f 100644
--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -34,7 +34,7 @@ Happy coding!
 1. Make sure the tests pass on your machine: `npm test`, note: these tests also apply the linter, so no need to lint separately
 1. Create a new branch: `git checkout -b my-branch-name`
 1. Make your change, add tests, and make sure the tests still pass
-1. Push to your fork and [submit a pull request][pr]
+1. Push to your fork and [submit a pull request][pr] (click "compare across forks")
 1. Pat your self on the back and wait for your pull request to be reviewed and merged.
 
 Here are a few things you can do that will increase the likelihood of your pull request being accepted:
~~~

### CODE REVIEW

**Comment by dscho** [comment]
>     * The link to the TODO was missing the '.md' extension so it
>       did not work.

Isn't it still missing that `.md` extension?

>     * The links for forking the repo and opening a PR did not work
>       because GitHub automatically adds '/blob/$branch' to relative links

That's too bad... But does it work if you prefix it with `../../`?

>  Should I add a mention of this link?

I think that would be a splendid idea!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
 # Contributing
 
-[fork]: /fork
-[pr]: /compare
+[fork]: https://github.com/gitgitgadget/gitgitgadget/fork
+[pr]: ../../compare
 [style]: https://palantir.github.io/tslint/
 [code-of-conduct]: CODE_OF_CONDUCT.md
+[TODO]: TODO.md
```
Comment (phil-blain) [inline CONTRIBUTING.md]: > Isn't it still missing that .md extension?

I put the extension in the list of links at the beginning of the file

**Comment by phil-blain** [comment]
> That's too bad... But does it work if you prefix it with ../../?

That indeed works! awesome. I updated by branch and added "(click "compare across forks")"

File: CONTRIBUTING.md (hunk 1 above)
Comment (dscho) [reply CONTRIBUTING.md]: Ah! Thanks for enlightening me.

**Comment by dscho** [comment]
Excellent, thank you so much, @phil-blain!

**Pull request merged**